Client/src/Components/DataCell/DataCell.js

```diff
@@ -29,6 +30,7 @@ export default function DataCell() {
             setOpen(true)
             return
         }
+        setAlertSuccess(true)

```

Comment:
Are we setting the alert if the user is able to input a file into the upload box or if the upload was successful?

Reply:
we are setting it to true if the file extension check passes (ie. file.json), so it will open the snackbar and display the success message on the page, then turn it to false.